neo4j-store/src/main/kotlin/org/ostelco/prime/storage/graph/Neo4jStore.kt

```diff
@@ -1317,12 +1319,17 @@ object Neo4jStoreSingleton : GraphStore {
                     will ensure that the invoice will be voided. */
                     createPurchaseRecord(customer.id, purchaseRecord)
                             .mapLeft {
-                                logger.error("Failed to save purchase record for customer ${customer.id}, invoice-id $invoiceId, invoice will be voided in Stripe")
-                                AuditLog.error(customerId = customer.id, message = "Failed to save purchase record - invoice-id $invoiceId, invoice will be voided in Stripe")
-                                BadGatewayError("Failed to save purchase record",
-                                        error = it)
+                                logger.error("Failed to save purchase record for customer ${customer.id}, invoice: $invoiceId, invoice will be voided in Stripe")
+                                AuditLog.error(customerId = customer.id,
+                                        message = "Failed to save purchase record - invoice: $invoiceId, invoice will be voided in Stripe")
+                                StorePurchaseError("Failed to save purchase record",
+                                        internalError = it)
                             }.bind()
 
+                    /* Adds purchase to customer history. */
+                    AuditLog.info(customerId = customer.id,
+                            message = "Purchased product $sku for ${product.price.amount} ${product.price.currency} (invoice: $invoiceId, charge-id: $chargeId)")
```

Comment:
`product.price.amount` is in lowest denomination (cents / øres). 
Do floating divide by 100.0 and then print up to 2 decimal places.